portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,22 +35,21 @@
 /** Servlet that returns some example content and handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  /** This class creates a storage for comments. 
-  *
-  */
+  /** Creates a storage for comments. */
   private class CommentStore {
     private String name;
     private String message; 
     private long timestamp;
   
-    CommentStore(String name, String message, long timestamp){
+    private CommentStore(String name, String message, long timestamp){
       this.name = name;
       this.message = message;
       this.timestamp = timestamp;
     }
   }
   // datastore can be accessed through the entire class. (global variable) 
-  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); 
+  // static access and won't be re-assigned.
+  static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); 
```

Comment:
```suggestion
  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); 
```
You still need private modifier because the scope just in this file.